src/main/java/ghbot/GhBot.java

```diff
@@ -0,0 +1,56 @@
+package ghbot;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import ghbot.exception.GhBotException;
+import ghbot.executor.ByeExecutor;
+import ghbot.executor.Executor;
+import ghbot.parser.Parser;
+import ghbot.storage.Storage;
+import ghbot.task.TaskList;
+import ghbot.ui.Ui;
+
+/**
+ * GhBot Class.
+ * This is the main class for the chatbot.
+ */
+public class GhBot {
+    private Storage storage;
+    private TaskList taskLst;
+    private Ui ui;
+
+    /**
+     * GhBot Constructor.
+     * @param filename The name of the file.
+     */
+    public GhBot(String filename) {
+        this.storage = new Storage(filename);
+        this.taskLst = new TaskList(storage.getInputFromFile());
+    }
+
+    /**
+     * You should have your own function to generate a response to user input.
```

Comment:
Is this meant to be the java documentation for this method? JavaDoc does not seem to be properly written. Perhaps, could remove this block.